anomaly_detector/storage/local_storage.py

```diff
@@ -81,10 +81,13 @@ class LSConfiguration(Configuration):
     """Configuration for local storage."""
 
     # Path to a file containing input data. If the value is '-' the input is expected on STDIN
-    LS_INPUT_PATH = Configuration.LOCAL_DATA_FILE #"validation_data/verification_data.json"
+    LS_INPUT_PATH = ""
     # Path to a file where the results will be stored. Results will be printed to STDOUT if empty
-    LS_OUTPUT_PATH = Configuration.LOCAL_RESULTS_FILE
+    LS_OUTPUT_PATH = ""
 
-    def __init__(self):
+    def __init__(self,config):
         """Initialize configuration."""
-        self.load()
+        self.load_from_env()
+        self.LS_INPUT_PATH=config.DATA_FILE
+        self.LS_OUTPUT_PATH = config.RESULTS_FILE
```

Comment:
What is this about? Why do you do this here?

Reply:
seems to be a mismatch of config variable and environment variable naming. Either use `DATA_FILE` or `LS_INPUT_PATH`.

Reply:
I've reverted to our original LOCAL_DATA_FILE and LOCAL_RESULTS_FILE